webcurator-ui/src/views/target/target-tabs/TargetTabPanelGeneral/TargetTabPanelGeneralSeeds.vue

```diff
@@ -84,50 +90,79 @@ watch(() => props.editing, async(newEditing) => {
       </div>
     </div>
 
-    <DataTable class="w-full" :rowHover="true" :value="targetSeeds.targetSeeds">
-      <Column field="seed" header="Seed">
-        <template #body="{ data }">
-          <div class="flex align-items-center justify-content-between">
+    <table class="w-full target-seed-parent-table">
+      <thead>
+        <tr style="border-bottom: 1px solid #E4E4E4;">
+          <th style="width: 20%; text-align: left; padding: .5rem;">Seed</th>
+          <th style="width: 10%; text-align: left; padding: .5rem;">Primary</th>
+          <th style="width: 15%; text-align: left; padding: .5rem;">Harvest Auth</th>
+          <th style="width: 15%; text-align: left; padding: .5rem;">Auth Agent</th>
+          <th style="width: 17.5%; text-align: left; padding: .5rem;">Start Date</th>
+          <th style="width: 17.5%; text-align: left; padding: .5rem;">End Date</th>
+          <th style="width: 10%; text-align: left; padding: .5rem;">Actions</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="(data, index) in targetSeeds.targetSeeds" :key="index">
```

Comment:
That's better to use data.id as the key. So it will not be affected by the order of the items.